gradle.properties

```diff
@@ -9,24 +9,24 @@
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 #
 # When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
+# This option should only be used with decoupled projects. For more details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Thu Mar 01 15:28:48 IST 2018
 org.gradle.jvmargs=-Xmx1536M
 android.enableBuildCache=true
 
-KOTLIN_VERSION=1.3.72
-BUTTERKNIFE_VERSION=10.1.0
-LEAK_CANARY_VERSION=1.6.2
-DAGGER_VERSION=2.21
-ROOM_VERSION=2.2.3
-PREFERENCE_VERSION=1.1.0
-CORE_KTX_VERSION=1.2.0
-ADAPTER_DELEGATES_VERSION=4.3.0
-PAGING_VERSION=2.1.2
-MULTIDEX_VERSION=2.0.1
-OKHTTP_VERSION=3.12.1
+KOTLIN_VERSION = 1.3.72
```

Comment:
I feel like there shouldn't be spaces before and after "=" sincerest of the file has no spaces.